app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/forms/DocumentationTypeFormMapping.scala

```diff
@@ -28,14 +28,21 @@ object DocumentationTypeFormMapping {
     "previousDocGoodsItemId" -> optional(text),
     "additionalInfoCode" -> optional(text),
     "additionalInfoDescription" -> optional(text),
-    "additionalDocCategoryCode" -> optional(text),
-    "additionalDocTypeCode" -> optional(text),
-    "additionalDocId" -> optional(text),
-    "additionalDocLPCO" -> optional(text),
-    "additionalDocName" -> optional(text),
+    "additionalDocument" -> seq(mapping(
+       "categoryCode" -> optional(text),
+       "typeCode" -> optional(text),
+       "id" -> optional(text),
+       "lpco" -> optional(text),
+       "name" -> optional(text),
+       "effectiveDateTime" -> optional(text),
+       "submitter" -> optional(text),
+       "writeOff" -> optional(text)
+    )(AdditionalDocumentType.apply)(AdditionalDocumentType.unapply)),
     "localReferenceNumber" -> optional(text),
-    "additionalDocPaymentID" -> optional(text),
-    "additionalDocPaymentCategory" -> optional(text),
-    "additionalDocPaymentType" -> optional(text)
+    "additionalPayment" -> seq(mapping(
```

Comment:
I can't see where these fields are mentioned on the ticket. Were the agreed with BAs?

Reply:
They're not. additionalPayment is the name given to the header level additional document.